streamx-common/src/main/scala/com/streamxhub/streamx/common/conf/CommonConfig.scala

```diff
@@ -46,6 +46,24 @@ object CommonConfig {
     classType = classOf[java.lang.String],
     description = "namespace for docker image used in docker building env and target image register")
 
+  val DOCKER_MAX_CONNECTIONS: ConfigOption = ConfigOption(
+    key = "streamx.docker.max-connections",
+    defaultValue = "100",
```

Comment:
Please define defaultValue directly as int type 100 and change the variable named `streamx.docker.max-connections-sec` to more visually tell the user the relevant time units.